ext/standard/basic_functions.stub.php

```diff
@@ -1722,14 +1722,14 @@ function array_pad(array $array, int $length, mixed $value): array {}
 
 /**
  * @return array<int|string, int|string>
- * @refcount 1
```

Comment:
Please avoid reordering changes. Another person will do the opposite.

Reply:
There were only a few inconsistencies, is it really better to keep two styles/order?  Later, someone else can point to these few lines to argue what the preffered style is..

Reply:
Can you please recheck? I belive the massive consistency is better vs. a few lines changed. Also please review the functional part, if the CS is fine, the PR is RTM. Thank you.